code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -1,4 +1,4 @@
-var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases, which were at one time hardcoded
+var/list/hardcoded_gases = list("o2","n2","co2","plasma", "freon") //the main four gases, which were at one time hardcoded
```

Comment:
I will make a new list, but using the above code, would I also need to make `/proc/meta_gas_list()
	. = new /list
	for(var/gas_path in subtypesof(/datum/gas))
		var/list/gas_info = new(4)
		var/datum/gas/gas = gas_path`

into `/proc/meta_gas_list()
	. = new /list
	for(var/gas_path in subtypesof(/datum/gas))
		var/list/gas_info = new(5)
		var/datum/gas/gas = gas_path`?